lib/MemoryModel/SymbolTableInfo.cpp

```diff
@@ -594,31 +591,6 @@ void SymbolTableInfo::dump()
     outs() << "}\n";
 }
 
-/*
- * Get the type size given a target data layout
- */
-u32_t SymbolTableInfo::getTypeSizeInBytes(const Type* type)
```

Comment:
Can we move these two functions to LLVMUtil?

Reply:
> 

You can try to fix this part and the CI might trigger once you have a new commit.

Reply:
Returned